builder/slide_and_fold_storage.cc

```diff
@@ -270,6 +270,7 @@ class slide_and_fold : public node_mutator {
     if (ff) {
       for (int d = 0; d < static_cast<int>(ff->size()); ++d) {
         expr overlap = (*ff)[d].second;
```

Comment:
This is pre-existing, but: on line 267, `auto ff` is bad. The auto is not the main issue, the issue is that it's making a copy of an std::vector. It should be a const reference: `const std::optional<std::vector<...>>&`

Reply:
Thanks, I just inlined it -  I think it's more readable like that.